protobuf/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM chatwork/alpine-sdk:3.11
+
+LABEL version="2.5.0"
+LABEL maintainer="akatsu@chatwork.com"
+
+ENV PROTOBUF_VERSION="2.5.0"
+
+RUN apk add --no-cache build-base curl automake autoconf libtool git zlib-dev && \
+    mkdir -p /protobuf && \
+    curl -L https://github.com/google/protobuf/archive/v${PROTOBUF_VERSION}.tar.gz | tar xvz --strip-components=1 -C /protobuf && \
+    cd /protobuf && \
+    autoreconf -f -i -Wall,no-obsolete && \
+    ./configure --prefix=/usr --enable-static=no && \
+    make -j2 && \
+    make install
```

Comment:
@awekuit 

I think setting `ENTRYPOINT` is better. 
Because the protobuf docker image is for executing the `protobuf` command.

```
docker run --rm -v `pwd`/protobuf/:/protobuf -v `pwd`/generated/:/generated chatwork/protobuf:2.5.0 /protobuf/[YOUR_PROTO_FILE].proto --java_out=/generated/ --proto_path=/protobuf/
```

Reply:
@cw-ozaki 
Surely. 
I fixed it!